package.json

```diff
@@ -31,9 +31,10 @@
     "react-clickdrag": "^2.0.0"
   },
   "devDependencies": {
+    "babelify": "^5.0.4",
     "brfs": "^1.4.0",
-    "insert-css": "^0.2.0",
-    "babelify": "^5.0.4"
+    "browserify": "^11.2.0",
```

Comment:
I added browserify because the demo script (`npm run-script demo`) doesn't work without a bundler installed and I tend not to install npm packages globally unless I have to -- I like to track the versions used alongside the project. I can remove it if it's not needed.